addons/im_livechat/static/src/embed/common/livechat_service.js

```diff
@@ -87,6 +87,7 @@ export class LivechatService {
         if (this.options?.force_thread) {
             this.state = SESSION_STATE.PERSISTED;
             this.thread = this.store.Thread.insert(this.options.force_thread);
+            this.thread.openChatWindow();
```

Comment:
I'm not against it, but why is this necessary?
The `force_thread` should have `open_chat_window` set to `True`.
But I'd be glad if we can do manual opens and get rid of `open_chat_window`.

Shouldn't this manual open only be done after chat hub restore? Is that already the case by luck?

Reply:
Yes it has `open_chat_window: true`, but the conversation could be folded. See test `chat request opens chat window`: there's chat bubble, and the `force_thread` is expected to open the chat window. The test needs this new LOC in order to pass with the change I made with `open_chat_window` that takes the mobile fixes and improvements into account.

Reply:
But didn't we determine that if it's folded, it should stay folded (cf. what is done inside open_chat_window) rather than forcing it open like here?

Reply:
I changed the `open_chat_window` to always openChatWindow in desktop, but in mobile there's some trickery to put in fold if it's not open.

So behaviourally fold stays fold only in mobile.

I don't know which is best, but this is not crucial and works at least as good as it's defined in test coverage.